src/test/scala/fi/oph/koski/raportit/AikuistenPerusopetuksenOppijamäärätRaporttiSpec.scala

```diff
@@ -0,0 +1,54 @@
+package fi.oph.koski.raportit
+
+import java.sql.Date.{valueOf => sqlDate}
+
+import fi.oph.koski.KoskiApplicationForTests
+import fi.oph.koski.henkilo.{LaajatOppijaHenkilöTiedot, MockOppijat}
+import fi.oph.koski.koskiuser.MockUser
+import fi.oph.koski.log.AuditLogTester
+import fi.oph.koski.organisaatio.MockOrganisaatiot.jyväskylänNormaalikoulu
+import fi.oph.koski.raportointikanta.RaportointikantaTestMethods
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+class AikuistenPerusopetuksenOppijamäärätRaporttiSpec extends FreeSpec with Matchers with RaportointikantaTestMethods with BeforeAndAfterAll {
+  private val application = KoskiApplicationForTests
+  private val raporttiBuilder = AikuistenPerusopetuksenOppijamäärätRaportti(application.raportointiDatabase.db, application.organisaatioService)
+  private lazy val raportti =
+    raporttiBuilder.build(List(jyväskylänNormaalikoulu), sqlDate("2010-01-01"))(session(defaultUser)).rows.map(_.asInstanceOf[AikuistenPerusopetuksenOppijamäärätRaporttiRow])
+
+  override def beforeAll(): Unit = loadRaportointikantaFixtures
+
+  "Aikuisten perusopetuksen oppijamäärien raportti" - {
+    "Raportti voidaan ladata ja lataaminen tuottaa auditlogin" in {
+      authGet(s"api/raportit/aikuistenperusopetuksenoppijamaaratraportti?oppilaitosOid=$jyväskylänNormaalikoulu&paiva=2010-01-01&password=salasana") {
+        verifyResponseStatusOk()
+        response.headers("Content-Disposition").head should equal(s"""attachment; filename="aikuisten_perusopetuksen_oppijamäärät_raportti-2010-01-01.xlsx"""")
+        response.bodyBytes.take(ENCRYPTED_XLSX_PREFIX.length) should equal(ENCRYPTED_XLSX_PREFIX)
+        AuditLogTester.verifyAuditLogMessage(Map("operation" -> "OPISKELUOIKEUS_RAPORTTI", "target" -> Map("hakuEhto" -> s"raportti=aikuistenperusopetuksenoppijamaaratraportti&oppilaitosOid=$jyväskylänNormaalikoulu&paiva=2010-01-01")))
+      }
+    }
+
+    "Raportin kolumnit" in {
+      lazy val r = findSingle(raportti)
+      println(r.toString)
+      r.oppilaitosNimi should equal("Jyväskylän normaalikoulu")
+      r.opetuskieli should equal("suomi")
+      r.oppilaidenMääräVOS should equal(5)
+      r.oppilaidenMääräMuuKuinVOS should equal(0)
```

Comment:
Mä tein nuorten perusopetuksen raporteille testicaseja niin että pystyin testaamaan että jokaiseen sarakkeeseen tulee joku muu arvo kuin 0. Näin ainakin varmistuu että kaikki data menee koko putken läpi onnistuneesti. Esim commitissa edda7fd66e47fbee681e0e9c7bdb0f413cdec440 oon lisännyt niitä fixtureja testiä varten.